Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -636,6 +636,13 @@ extension BrowserViewController: TabManagerDelegate {
     func tabManager(tabManager: TabManager, didSelectedTabChange selected: Browser?, previous: Browser?) {
         if let wv = selected?.webView {
             webViewContainer.addSubview(wv)
+            if let url = wv.URL?.absoluteString {
+                profile.bookmarks.isBookmarked(url, success: { bookmarked in
+                    self.toolbar.updateBookmarkStatus(bookmarked)
+                    }, failure: { err in
```

Comment:
Nit: indentation is kinda funky here (the line above is on the same level as this brace, and the closing brace below is also misaligned)